src/web/src/components/Banner.tsx

```diff
@@ -4,81 +4,81 @@ import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
-      color: theme.palette.text.primary,
-      fontWeight: 'bold',
-      fontSize: '12rem',
-      display: 'block',
-      top: theme.spacing(20),
-      left: theme.spacing(15),
-      [theme.breakpoints.between('xs', 'sm')]: {
-        fontSize: '4rem',
-        textAlign: 'left',
-        left: theme.spacing(4),
-        right: theme.spacing(4),
-        top: theme.spacing(14),
-      },
-      [theme.breakpoints.between('md', 'lg')]: {
-        fontSize: '8rem',
-      },
-      [theme.breakpoints.up('xl')]: {
-        fontSize: '12rem',
-      },
-    },
     heroBanner: {
       maxHeight: '100vh',
       overflow: 'hidden',
       position: 'relative',
     },
+    textsContainer: {
+      position: 'absolute',
+      width: '100%',
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
+      alignItems: 'center',
+      top: '40%',
+    },
+    telescopeTitle: {
+      color: '#A0D1FB',
+      fontWeight: 'bold',
+      fontSize: 'clamp(2.5rem, 4.2vw, 6em)',
+      letterSpacing: '.45em',
+      display: 'block',
+      zIndex: 1000,
+      marginBottom: '10px',
+    },
+    quoteText: {
+      width: '70%',
+      color: '#FFFFFF',
+      fontSize: 'clamp(.7rem, .95vw, 1.6em)',
+      display: 'block',
+      textAlign: 'center',
+      zIndex: 1000,
+    },
     version: {
       position: 'absolute',
       opacity: 0.85,
-      bottom: theme.spacing(6),
-      left: theme.spacing(15),
-      fontSize: '1.75rem',
-      color: theme.palette.text.primary,
-      [theme.breakpoints.between('xs', 'sm')]: {
-        left: theme.spacing(4),
-        right: theme.spacing(4),
-        fontSize: '1.75rem',
-      },
-      [theme.breakpoints.between('md', 'lg')]: {
-        fontSize: '2rem',
-      },
-      [theme.breakpoints.up('xl')]: {
-        fontSize: '4rem',
-      },
+      bottom: theme.spacing(3),
+      left: theme.spacing(3),
+      fontSize: '1.3em',
+      color: 'white',
       textDecorationLine: 'none',
       '&:hover': {
         textDecorationLine: 'underline',
       },
     },
     icon: {
-      height: '5.6rem',
-      width: '5.6rem',
-      position: 'relative',
-      bottom: theme.spacing(20),
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'center',
+      width: '100%',
+      position: 'absolute',
+      bottom: theme.spacing(1),
       zIndex: 300,
       margin: '0 auto',
-      [theme.breakpoints.down(1080)]: {
-        left: '55%',
-        bottom: theme.spacing(18),
+      '& .MuiFab-root': {
+        backgroundColor: 'transparent',
+        boxShadow: 'none',
+        display: 'grid',
+      },
+      [theme.breakpoints.down(1024)]: {
+        marginBottom: '60px',
       },
     },
     anchor: {
       position: 'relative',
       top: '1rem',
     },
     container: {
-      height: 0,
+      bottom: '0',
     },
     arrowDownIcon: {
-      color: theme.palette.text.primary,
-      fontSize: '3rem',
+      color: 'white',
```

Comment:
Let's add this colour to the theme

Reply:
I don't want to connect the landing page with the theme colors now, let's make this when implementing a dark/light themes in the landing page. For now, let's land the new land page without a dark/light theme.